Source/Tests/TestHarnessRunner.cpp

```diff
@@ -115,31 +115,33 @@ class TestEnvLoader final : public FEXCore::Config::Layer {
 }
 
 int main(int argc, char **argv, char **const envp) {
+  FEXCore::Allocator::SetupFaultEvaluate();
   LogMan::Throw::InstallHandler(AssertHandler);
   LogMan::Msg::InstallHandler(MsgHandler);
   FEXCore::Config::Initialize();
-  FEXCore::Config::AddLayer(std::make_unique<FEX::ArgLoader::ArgLoader>(argc, argv));
+  FEXCore::Config::AddLayer(fextl::make_unique<FEX::ArgLoader::ArgLoader>(argc, argv));
   FEXCore::Config::AddLayer(FEXCore::Config::CreateEnvironmentLayer(envp));
   FEXCore::Config::Load();
 
   auto Args = FEX::ArgLoader::Get();
 
   if (Args.size() < 2) {
     LogMan::Msg::EFmt("Not enough arguments");
+    FEXCore::Allocator::ClearFaultEvaluate();
```

Comment:
Why don't we use a scope guard for this? Are there exit paths where we *don't* want to call this function?

Reply:
Changed over to a new scope guard. Resolved.